### ISSUE TITLE
MobileNet v2 training options

### ISSUE THREAD

**Issue by lld533** [d-li14/mobilenetv2.pytorch#2]
Could you please kindly share your training options for MobileNet v2 on ImageNet when top1 accuracy finally reaches 72.0%? Thanks a lot!

**Comment by mathmanu** [comment]
Could you please share your training hyper parameters? Number of epochs, learning rate, learning rate schedule, weight decay etc.

**Comment by lld533** [comment]
I trained MobileNet V2 from scratch by calling
python3 imagenet.py -d /path/to/your/ImageNet/root/ -j16 --epochs=300 --arch="mobilenetv2" --gpu-id="0,1,2,3" --lr=0.045 --lr-decay="linear2exp" --gamma=0.98 --weight-decay=0.00004

Now, the best model achieves 71.79% top-1 accuracy at Epoch 249. The training is expected to finish in a few days. Hardware environment: 4-way 2080ti, Software environment: Pytorch 1.0.

**Comment by mathmanu** [comment]
Thankyou

**Comment by mehtadushy** [comment]
> 
> 
> I trained MobileNet V2 from scratch by calling
> python3 imagenet.py -d /path/to/your/ImageNet/root/ -j16 --epochs=300 --arch="mobilenetv2" --gpu-id="0,1,2,3" --lr=0.045 --lr-decay="linear2exp" --gamma=0.98 --weight-decay=0.00004
> 
> Now, the best model achieves 71.79% top-1 accuracy at Epoch 249. The training is expected to finish in a few days. Hardware environment: 4-way 2080ti, Software environment: Pytorch 1.0.

Could you please make this a part of the ReadMe ? I was about to open the same issue to ask for the hyperparameter details when I noticed this closed issue.
